sdk/monitor/azure-monitor-opentelemetry-exporter/azure/monitor/opentelemetry/exporter/export/_base.py

```diff
@@ -162,6 +173,24 @@ def __init__(self, **kwargs: Any) -> None:
 
             collect_statsbeat_metrics(self)
 
+        # Initialize customer statsbeat if enabled
+        self._customer_statsbeat_metrics = None
```

Comment:
Any reason why we set `self._customer_statsbeat_metrics` to `None` and not just directly initialize it? This would save a lot of code throughout the SDK for checking for `None` type.